### PR TITLE
QAE-343 - E2E test case for transparent header disable on complete website

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for transparent header disable on complete website

### Screenshots

https://share.getcloudapp.com/X6u82Zme

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import { createURL,	createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:1]: "publish-post" importing path

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import { setCustomize } from '../../../../utils/customize';
+describe( 'transparent header in the customizer', () => {
+	it( 'disable on archive, 404 and search page should apply corectly', async () => {
+		const Disableonarchive = {
+			'transparent-header-enable': 1,
+			'transparent-header-disable-archive': 1,
+		};
+		await setCustomize( Disableonarchive );
+		await page.goto( createURL( '/author/admin' ), {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:10]: incorrect url

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableArchive = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.ast-theme-transparent-header #masthead' ) );
+		await expect( disableArchive ).toBeNull( );
+
+		await setCustomize( Disableonarchive );
+		await page.goto( createURL( '/12' ), {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:18]: @SimranChougule1 getting console error for this url "/12" please take a look

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
+import { createURL,	createNewPost } from '@wordpress/e2e-test-utils';
+import { publishPost } from '../../../../utils/publish-post';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Transparent header in the customizer', () => {
+	it( 'disable on archive, 404 and search page should apply corectly', async () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:5]:
```suggestion
	it( 'disable on archive, 404 and search page should apply correctly', async () => {
```

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+			waitUntil: 'networkidle0',
+		} );
+		await page.click( '#wp-block-search__input-1' );
+		await page.keyboard.type( 'test' );
+		await page.keyboard.press( 'Enter' );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableSearch = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.ast-theme-transparent-header #masthead' ) );
+		await expect( disableSearch ).toBeNull( );
+	} );
+	it( 'disable on blog page should apply corectly', async () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:43]:
```suggestion
	it( 'disable on blog page should apply correctly', async () => {
```

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+			ppStatus = await publishPost();
+		}
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableLatestPost = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.ast-theme-transparent-header #masthead' ) );
+		await expect( disableLatestPost ).toBeNull( );
+	} );
+	it( 'disable on pages should apply corectly', async () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:81]:
```suggestion
	it( 'disable on pages should apply correctly', async () => {
```

File: tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+			ppStatus = await publishPost();
+		}
+		await page.goto( createURL( '/test' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disablePages = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.ast-theme-transparent-header #masthead' ) );
+		await expect( disablePages ).toBeNull( );
+	} );
+	it( 'disable on posts should apply corectly', async () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/disable-on-complete-website.test.js:99]:
```suggestion
	it( 'disable on posts should apply correctly', async () => {
```

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**